Makefile

```diff
@@ -14,7 +14,10 @@ all: lint test
 
 COV_REPORT := overalls.coverprofile
 
-DOCKER_IMAGE := uber/fx
+DOCKER_MAKE_CMD ?= test
```

Comment:
I actually dislike having separate build files like this, it makes things less explicit, I think it's fine here, not much logic and it directly links to logic in the Makefile